CONTRIBUTING.md

```diff
@@ -7,6 +7,8 @@ yarn
 yarn test
 ```
 
+Run `yarn prettier path/to/file.php` to format a single file.
```

Comment:
If we add `yarn`, we also should add `npm` instruction

Reply:
Well, the repo only has a `yarn.lock` file so it's pretty much set up to work with yarn. But I'm happy to add `npm` instructions as well, it might just cause problems for people then they don't get locked depency versions.

Reply:
@Epskampie npm support `package-lock.json` since `npm@5`, but how it is related to issue?

Reply:
Yeah, I know it does, but there is no `package-lock.json` present in the repo, look: https://github.com/prettier/plugin-php

Anyway, I added npm instructions!

Reply:
@evilebottnawi I think for contributing to the plugin, we should enforce the usage of yarn over npm. Prettier core does the same ([see here](https://github.com/prettier/prettier/blob/master/CONTRIBUTING.md))

Reply:
No worries, it's reverted!